telegraf/rootfs/Dockerfile

```diff
@@ -3,11 +3,14 @@ FROM quay.io/deis/base:v0.3.4
 COPY . /
 
 RUN mkdir -p /usr/local/bin/ \
-	&& curl -SL -o /tmp/telegraf.deb https://dl.influxdata.com/telegraf/releases/telegraf_1.0.0_amd64.deb \
-    && dpkg -i /tmp/telegraf.deb \
-    && rm /tmp/telegraf.deb \
-    && chmod +x /usr/bin/telegraf \
+	# && curl -SL -o /tmp/telegraf.deb https://dl.influxdata.com/telegraf/releases/telegraf_1.0.0_amd64.deb \
```

Comment:
i didnt want to remove these because we will be going back to using the package once they release my new plugin.